src/components/mixins/Messages.ts

```diff
@@ -24,7 +24,7 @@ export interface MessagesMixin extends Vue {
 	},
 } )
 export default class Messages extends ( Vue as VueConstructor<MessagesMixin> ) {
-	public MESSAGE_KEYS = MessageKeys;
+	public readonly MESSAGE_KEYS = MessageKeys;
```

Comment:
If we want it as const, it should walk like a const

Reply:
(not part of this PR really)
We both agreed with your opinion when doing this change, the current state is a result of what is technically possible AFAIK. https://github.com/vuejs/vue/issues/6004